config/crd/patches/webhook_in_bmcjobs.yaml

```diff
@@ -2,7 +2,7 @@
 apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
-  name: powercyclejobs.bmc.tinkerbell.org
```

Comment:
Why do we have patches for a new spec?

Reply:
I believe these files were auto-generated by `kubebuilder create api`?
@micahhausler Do you have more context on what they are used for?

Reply:
The probably are, but that doesn't matter much. Consolidate by regenerating everything ;).